app/src/main/res/values/configuration.xml

```diff
@@ -5,4 +5,5 @@
 <resources>
     <!-- We are overriding this value in values-sw600dp -->
 
+    <integer name="channelTileAnimTime">100</integer>
```

Comment:
I like this timing better: nice! What's interesting is that it's so fast that it's not super noticeable that it's animating (when compared to the translation animations) but it gets serves the intended effect of emphasizing the selected item.

It might be slightly too fast (I think it completes before translation does) but I think this is good enough. :)